agent/src/main/java/com/code_intelligence/jazzer/api/Jazzer.java

```diff
@@ -23,68 +23,83 @@
  * Helper class with static methods that interact with Jazzer at runtime.
  */
 final public class Jazzer {
-  private static Class<?> jazzerInternal = null;
+  private static final Class<?> jazzerInternal;
```

Comment:
According to Google's style guide, static final fields should use upper shake case https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names.

Reply:
Thanks, this also neatly solves the naming problem.